src/chrome/content/rules/POLi-Payments.xml

```diff
@@ -0,0 +1,103 @@
+<!--
+	Non-functional subdomains:
+		- polipayments.com
+			- status.apac	(m)
+			- autodiscover	(r)
+			- www.polipayments.comwww	(m)
+			- dd	(m)
+			- demo.dev	(m)
+			- consoles.dev1	(m)
+			- email	(m)
+			- forum	(m)
+			- jenkinsr2	(m)
+			- news	(m)
+			- play	(m)
+			- merchantapi.rega-staging	(m)
+			- www.stg1	(m)
+			- www.systemcheck	(m)
+			- systemscheck	(m)
+			- test	(m)
+			- consoles.test01	(m)
+			- www.merchant.test01	(m)
+			- ww.w	(m)
+			- whatever	(m)
+			- ww2w	(m)
+
+		- paywithpoli.com	(t)
+			- www	(t)
+			- polihealth-ext.a2	(t)
+			- express.apac	(m)
+			- apac-test	(r)
+			- nz10100.apac-test	(r)
+			- nz10101.apac-test	(r)
+			- au	(t)
+			- commonapi-int.dev1	(t)
+			- multibankapi.dev1	(t)
+			- www.tst1	(s)
+			- au00710.tst1	(s)
+			- consoleapi-ext.tst1	(s)
+			- txn.tst1	(s)
+			- wiki.tst1	(s)
+
+		- polipay.co.nz	(m)
+			- blog	(m)
+			- merchants	(s)
+			- www.merchants	(s)
+			- transact	(s)
+			- www.transact	(s)
```

Comment:
`merchants` and `transact` have a valid cert, but serve an incomplete cert chain.